torch/_torch_docs.py

```diff
@@ -2339,6 +2339,30 @@ def merge_dicts(*dicts):
     tensor(7)
 """)
 
+add_docstr(torch.vdot,
+           r"""
+vdot(input, other) -> Tensor
+
+Computes the dot product (inner product) of two tensors.
+
+.. note:: This function does not :ref:`broadcast <broadcasting-semantics>`.
+
+.. note:: The vdot(a, b) function handles complex numbers differently than dot(a, b).
+          If the arguments are complex, the complex conjugate of the first argument
+          is used for the calculation of the dot product.
+
```

Comment:
Something like:

Args:
  input (Tensor): first tensor in the dot product. Its conjugate is used if it's complex.
  other (Tensor): second tensor in the dot product.

Keyword args:
  {out}

Reply:
Thanks!!!
 Done.